twine/auth.py

```diff
@@ -1,12 +1,12 @@
-import warnings
-import getpass
 import functools
-from typing import Optional, Callable
+import getpass
+from typing import Callable, Optional
+import warnings
 
 import keyring
 
-from . import utils
 from . import exceptions
```

Comment:
We seem to have a mixture of absolute imports `from twine import ...` and absolute-relative imports `from . import ...` we should probably standardize on those

Reply:
Happy to address this after we settle on one these PRs.